### ISSUE TITLE
Arcane Blink does not modify the values for some spells

### ISSUE THREAD

**Issue by GathX** [ValveSoftware/Dota2-Gameplay#6555]
### Item name

Arcane Blink

### Description

Arcane Blink does not seem to modify the duration of slow that have DOTs like Venomous Gale, and neither does the bonus from Rubick's Spell Steal in the tests that I have ran. This is quite interesting because individual slows like Cinder Brew DO get extended by debuff amplifications. Warlock's Shadow word, being a DOT that applies no other conditions, is also unaffected. Nonetheless, Doom's Doom does seem to be affected by it.

Long story short, debuff amplification seems to work irregularly across spells. I honestly do not understand what is considered a "debuff" for this condition.

### Example Match ID and Timestamp

None, everything can be recreated in demo.

**Comment by GathX** [comment]
For further clarification, I am just talking about the 20% extra duration not being applied. I know for certain that it doesn't apply to Venomous Gale, Poison Nova, Shadow Strike, or Shadow Word. Although, I am not sure if the last one should be covered by it.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/5840,

https://dota2.fandom.com/wiki/Status_resistance

TLDR, status resistance or debuff manipulation affects spells differntly.

For Venomous Gale, Shadow Strike, it reduces the slow amount
For Doom, duration is reduced (damage done is the same, as damage interval is reduced)
For Shadow word, Poison Nova, it is uneffected.
